### PR TITLE
More boilerplate

### PR DESCRIPTION
**Pull request by Awjin**

### CODE REVIEW

File: tool/embedded-protocol.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+const execSync = require('child_process').execSync;
+const https = require('https');
+
+const PROTO_NAME = 'embedded_sass.proto';
+
+/**
+ * Downloads the latest version of the Embedded Protocol into the given
+ * directory.
+ */
+function downloadProto(dir, callback) {
```
Comment (nex3) [inline tool/embedded-protocol.js:15]: You could probably clean this up by using https://www.npmjs.com/package/node-fetch with `async`/`await` (at the cost of adding another dev dependency).

File: tool/post-install.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+// Create the vendor directory.
+const dir = 'dist/lib/src/vendor';
+mkdir(dir, {recursive: true}, error => {
+  if (error) {
+    throw Error(`Could not create vendor directory.\n${error.message}`);
+  }
+});
+
+// Download and compile the Embedded Sass Protocol.
+protocol.downloadProto(dir, () => protocol.writePbjs(dir));
```
Comment (nex3) [inline tool/post-install.js:17]: Why not write the declaration file as well?

File: tool/post-install.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+const mkdir = require('fs').mkdir;
+const protocol = require('./embedded-protocol');
+
+// Create the vendor directory.
+const dir = 'dist/lib/src/vendor';
+mkdir(dir, {recursive: true}, error => {
```
Comment (nex3) [inline tool/post-install.js:10]: Consider using the `fs.promises` API along with top-level `await`.

File: tool/embedded-protocol.ts [hunk 4]
```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import {execSync} from 'child_process';
+import {promises} from 'fs';
```
Comment (nex3) [inline tool/embedded-protocol.ts:6]:
```suggestion
import {promises as fs} from 'fs';
```

This'll get you cleaner-looking invocation sites :).

Comment (Awjin) [reply tool/embedded-protocol.ts]: Thanks!

**Pull request merged**